ccmlib/common.py

```diff
@@ -76,6 +77,36 @@ class UnavailableSocketError(CCMError):
     pass
 
 
+class LogPatternToVersion(object):
+    def __init__(self, patterns_to_versions):
+        assert type(patterns_to_versions) is dict, "LogPatternToVersion takes a dictionary."
+        self.versions_to_patterns = patterns_to_versions
```

Comment:
Why the name change here? I think the argument name should be `version_to_patterns` here.